kubernetes/cronjob/cronjob.yaml

```diff
@@ -31,6 +31,14 @@ spec:
               requests:
                 cpu: "500m"
                 memory: "256Mi"
+            livenessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /healthz
+                port: 10258
+                scheme: HTTPS
+              initialDelaySeconds: 3
+              periodSeconds: 3
```

Comment:
Isn't `periodSeconds` of 3 too aggressive given that min interval is 1 minute?

Reply:
Which `periodSeconds` you prefer? 30 seconds is better or?

Reply:
@a7i would you point me to the place where you see the 1 minute interval?

Reply:
The lowest cronjob interval you can have is 1 minute.

Reply:
IMO, we should just remove it to use the default value of 10 seconds

Reply:
In the descheduler case there's no significant different between 3s and 10s. Looks fine to me use the default value.

Reply:
It seems we align this comment and not need to update, so may I click resolve conversation?

Reply:
> Looks fine to me to use the default value in all the cases.

Basically removing `periodSeconds` should default to 10.

Reply:
Already change  `periodSeconds` to 10, I will click "Resolve conversation"